Rules/CommonScripts/Report.as

```diff
@@ -3,21 +3,35 @@
 
 //time (in seconds) between repeated reports
 const u32 reportRepeatTime = 1 * 60;
+u8 nonModTeam=0; //Sandbox' default team.
 
 const SColor reportMessageColor(255, 255, 0, 0);
 
 void onInit(CRules@ this)
 {
 	this.addCommandID("notify");
 	this.addCommandID("report");
+	this.addCommandID("mod_team");
 }
 
 bool onClientProcessChat(CRules@ this, const string& in text_in, string& out text_out, CPlayer@ player)
 {
 	if((text_in == "!moderate" || text_in == "!m") && player.isMod())
 	{
-		moderate(this, player);
-
+		if(this.gamemode_name=="CTF"||this.gamemode_name=="TTH")
```

Comment:
I don't think you should be hardcoding specific game modes. Maybe there's a way to check if the game mode has a spectator team.